src/main/webapp/i18n/de/global.json

```
@@ -72,6 +72,10 @@
                 "metis": {
                     "post": "Beitrag",
                     "answerPost": "Antwort"
+                },
+                "messages": {
+                    "conversation": "Gespräch",
```

```suggestion
                    "conversation": "Konversation",
```
"Gespräch" is more related to verbal conversations.

---

thanks for your suggestion 😊